openstef/model/regressors/better_dazls.py

```diff
@@ -0,0 +1,156 @@
+import numpy as np
+from sklearn.metrics import r2_score, mean_squared_error
+from sklearn.preprocessing import MinMaxScaler
+from sklearn.neighbors import KNeighborsRegressor
+from sklearn.utils import shuffle
+from sklearn.base import BaseEstimator
+
+
+# DAZLS algorithm
+class BetterDazls(BaseEstimator):
+    """
+    The model carries out wind and solar power prediction for unseen target substations using training data from other substations with
+    known components.
+
+    Any data-driven model can be plugged and used as the base for the domain and the adaptation model.
```

Comment:
Nice docstring. Is there already a link available with more elaborate documentation on the model?

Reply:
The scientific paper has more information about the model but it is not publicly available yet.